contracts/TopShot.cdc

```diff
@@ -455,9 +456,101 @@ pub contract TopShot: NonFungibleToken {
 
     }
 
+    // This is an implementation of a custom metadata view for Top Shot.
+    // This view contains the play metadata.
+    //
+    pub struct TopShotMomentsMetadataView {
+
+        pub let fullName: String?
+        pub let firstName: String?
+        pub let lastName: String?
+        pub let birthdate: String?
+        pub let birthplace: String?
+        pub let jerseyNumber: String?
+        pub let draftTeam: String?
+        pub let draftYear: String?
+        pub let draftSelection: String?
+        pub let draftRound: String?
+        pub let teamAtMomentNBAID: String?
+        pub let teamAtMoment: String?
+        pub let primaryPosition: String?
+        pub let height: String?
+        pub let weight: String?
+        pub let totalYearsExperience: String?
+        pub let nbaSeason: String?
+        pub let dateOfMoment: String?
+        pub let playCategory: String?
+        pub let playType: String?
+        pub let homeTeamName: String?
+        pub let awayTeamName: String?
+        pub let homeTeamScore: String?
+        pub let awayTeamScore: String?
+        pub let seriesNumber: String?
```

Comment:
is there a reason they are all string types? For instance series is a `UInt32` in the contract

Reply:
Agreed, series number, play ID, set ID, serial number, and the number of moments in this moment's edition should all be included in this, but should be integers and not strings. 👍

Reply:
Got it. PR is updated with these changes. Thanks.